packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -191,33 +201,80 @@ contract DependencyRegistryV0 is
     }
 
     /**
-     * @notice Updates script for dependencyType `_dependencyType` at script ID `_scriptId`.
-     * @param _dependencyType dependency to be updated.
-     * @param _scriptId Script ID to be updated.
+     * @notice Updates script for dependencyType `_dependencyType` at script `_index`,
+     *         by way of providing a string to write to bytecode storage.
+     * @param _dependencyType Name of dependency type (i.e. "type@version") used to identify dependency.
+     * @param _index The index of a given script, relative to the overall `scriptCount`.
      * @param _script The updated script value. Required to be a non-empty
-     * string, but no further validation is performed.
+     *                string, but no further validation is performed.
      */
     function updateDependencyScript(
         bytes32 _dependencyType,
-        uint256 _scriptId,
+        uint256 _index,
         string memory _script
     ) external {
         _onlyAdminACL(this.updateDependencyScript.selector);
         _onlyNonEmptyString(_script);
         _onlyExistingDependencyType(_dependencyType);
         Dependency storage dependency = dependencyDetails[_dependencyType];
-        require(_scriptId < dependency.scriptCount, "scriptId out of range");
+        _onlyInRangeIndex(_index, dependency.scriptCount);
```

Comment:
nit: using named args might be appropriate here (for all uses in this contract) since we have two adjacent values with the same type

Reply:
Good call out – updated per your suggestion in latest commit.